Get rid of flash type, version, revision, VisionModule and CPU arch since we don't have versioninfo.bb
and we don't use a revision number.
Also we known it's a normal image and not a small flash image.
Use the old way to display version number (after image name).
Add 'unofficial'  to image name to make it clear it's not an official Open Vision image.
CPU-arch already displayed in CPU info (duplicate).
Restore time of update.

diff --git a/lib/python/Screens/About.py b/lib/python/Screens/About.py
index f960464fe..c5a97add7 100644
--- a/lib/python/Screens/About.py
+++ b/lib/python/Screens/About.py
@@ -29,15 +29,11 @@ class About(Screen):
 		cpu = about.getCPUInfoString()
 		AboutText += _("CPU: ") + cpu + "\n"
 		AboutText += _("CPU brand: ") + about.getCPUBrand() + "\n"
-		AboutText += _("CPU architecture: ") + about.getCPUArch() + "\n"
-		AboutText += _("Flash type: ") + about.getFlashType() + "\n"
-		AboutText += _("Image: ") + about.getImageTypeString() + "\n"
-		AboutText += _("Open Vision version: ") + about.getVisionVersion() + "\n"
-		AboutText += _("Open Vision revision: ") + about.getVisionRevision() + "\n"
-		AboutText += _("Open Vision module: ") + about.getVisionModule() + "\n"
+		AboutText += _("Image: ") + about.getImageTypeString() + " (unofficial)\n"
 		AboutText += _("Build date: ") + about.getBuildDateString() + "\n"
-		AboutText += _("Last upgrade: ") + about.getUpdateDateString() + "\n"
-
+		ImageVersion = _("Last upgrade: ") + about.getImageVersionString()
+		self["ImageVersion"] = StaticText(ImageVersion)
+		AboutText += ImageVersion + "\n"
 		# [WanWizard] Removed until we find a reliable way to determine the installation date
 		# AboutText += _("Installed: ") + about.getFlashDateString() + "\n"
 
